src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -113,65 +184,62 @@ private static bool IsCodeActionNotSupportedByLSP(IUnifiedSuggestedAction sugges
             }
 
             var codeAction = suggestedAction.OriginalCodeAction;
-            currentTitle += codeAction.Title;
 
-            var diagnosticsForFix = GetApplicableDiagnostics(request.Context, suggestedAction);
+            if (!isMainCodeAction)
+            {
+                currentTitle += codeAction.Title;
```

Comment:
titles cannot be concatenated like this.  loc strings don't work that way in all languages.